examples/hardhat/hardhat.config.js

```diff
@@ -1,6 +1,6 @@
 require("hardhat-deploy");
 
-const {PK, INFURA_KEY} = process.env;
+const {PK, INFURA_PROJECT_ID} = process.env;
```

Comment:
Now this is an interesting change, because it makes so much sense (since that is literally what it is), although I suspect this will take some time for people to adjust to - including myself.

Reply:
To be fair, this change was made just to be consistent with truffle project 😉

Reply:
Yes of course and this is why I say it makes so much sense.